spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -406,12 +406,28 @@
             'documents-form-documents-field',
             'spec/fixtures/files/blank_job_spec.pdf'
           )
+
+          click_on 'Save and continue'
+
+          fill_in_application_details_form_fields(vacancy)
+          click_on 'Save and continue'
+
+          expect(page).to have_content(I18n.t('jobs.review'))
+          expect(page).to_not have_content('No documents uploaded')
+
+          verify_all_vacancy_details(vacancy)
```

Comment:
Is there a way to cut down the number of times we call this? There is [a ticket](https://dfedigital.atlassian.net/secure/RapidBoard.jspa?rapidView=21&view=planning&selectedIssue=TEVA-511&issueLimit=100&search=refactor) to look at moving away from using this black box full of expectations, prompted by [this comment](https://github.com/DFE-Digital/teacher-vacancy-service/pull/1335#discussion_r382050236) from @tatyree.

What's the argument for calling it in more than 0 of these two tests?

Reply:
This is just the code Renny wrote, I'm happy for the `verify_all_vacancy_details` call to be omitted since we're only testing the supporting documents section here